nUpdate.Shared/Updating/UpdateResult.cs

```diff
@@ -47,6 +47,48 @@ internal class UpdateResult
                         config.Architecture == Architecture.X64 && !is64Bit)
                         continue;
 
+
+                    //Only check conditions if the remote conditions are not null or collection is empty
+                    if (config.Rolloutconditions != null && config.Rolloutconditions.Count != 0)
+                        //If the local conditions are null or the collection is empty then continue
+                        if (conditions == null || !conditions.Any())
```

Comment:
Maybe we could use the "?"-operator here.

Reply:
Not with Any() 
I can write  !conditions?.Count() = 0